core/sdk-core/pom.xml

```diff
@@ -211,6 +211,24 @@
                     </properties>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>bundle-manifest</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>manifest</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <instructions>
+                        <Require-Capability>osgi.serviceloader; filter:="(|(osgi.serviceloader=software.amazon.awssdk.http.SdkHttpService)(osgi.serviceloader=software.amazon.awssdk.http.SdkAsyncHttpService))";cardinality:=multiple,osgi.extender; filter:="(osgi.extender=osgi.serviceloader.processor)"</Require-Capability>
```

Comment:
Do we need the "|" symbol before first "osgi.serviceloader"?

We can also just put ` osgi.extender; filter:="(osgi.extender=osgi.serviceloader.processor)"` which will allow sdk-core to load any service instead of limiting to SdkHttpService and SdkAsyncHttpService

Reply:
The `|` is a logical OR so customers don't need to load both types of HTTP clients if they only intend to use one of them.

> We can also just put osgi.extender; filter:="(osgi.extender=osgi.serviceloader.processor)" which will allow sdk-core to load any service instead of limiting to SdkHttpService and SdkAsyncHttpService

What's the benefit to this? This is the only use of `ServiceLoader` in `core` afaik

Reply:
Yep no additional advantage now. Might be useful in case if there is an additional service interface and we don't need to remember to update the pom. I am okay with current change too.